### ISSUE TITLE
How to create Custome List View element in List View

### ISSUE THREAD

**Issue by kunjee17** [fsprojects/Fabulous#273]
I was trying to create Custom List View Element but It seems it is not possible as things are wrapped in Content Cell. Is it possible to by pass that or any other work around to create Custom List Element. 

Was trying to create a list with Stack Layout in it but didn't work out.

**Comment by TimLariviere** [comment]
Not sure if I understand correctly.

Do you mean you want to use a `StackLayout` inside each element?
If so, you can already do it. Our `ViewCell` implementation accepts any `ViewElement`, even layouts like `StackLayout`.
(... and you can't replace our implementation without doing a lot of work unfortunately)

```
View.ListView(
      items=[
            View.StackLayout(
                  children=[
                       View.Label(text="")
                       View.Label(text="")
                  ])

            View.StackLayout(
                  children=[
                       View.Label(text="")
                       View.Label(text="")
                  ])
      ])
```

**Comment by kunjee17** [comment]
@TimLariviere yes. They are. When you have two cells they are over lapping each other. :( . Any ways to solve it?

**Comment by TimLariviere** [comment]
You can set  `hasUnevenRows=True` and `rowHeight=-1` on your ListView.
This will disable an accurate virtualization (you might see the scroll bar jumping when scrolling to the end) but it should solve your problem.

**Comment by kunjee17** [comment]
Thanks @TimLariviere  it did solve the issue. Sorry I can close the issue as it is already closed. :)
